sdks/python/src/honcho/session_context.py

```
@@ -0,0 +1,145 @@
+            assistant: The assistant peer (Peer object or peer ID string) to use
+                       for determining message roles. Messages from this peer will
+                       be marked as "assistant", others as "user"
+
+        Returns:
+            A list of dictionaries in OpenAI format, where each dictionary contains
+            "role" and "content" keys suitable for the OpenAI API
+
+        Raises:
+            ValidationError: If assistant parameter is invalid
+        """
+        assistant_id = assistant.id if isinstance(assistant, Peer) else assistant
```

Pylint (3.3.7)

[error] Using variable 'Peer' before assignment

(E0601)